### PR TITLE
Updated backend dependencies

### PR DESCRIPTION
**Pull request by javiereguiluz**
Dependencies were updated with Composer 2 RC1.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -35,16 +35,13 @@ jobs:
   include:
     # Run tests with the dependencies from composer.lock
     - php: 7.2
-    - php: 7.3
```
Comment (stof) [inline .travis.yml:38]: so no tests on PHP 7.3 anymore ?

Comment (javiereguiluz) [reply .travis.yml:38]: No ... I thought that the lowest supported version (7.2) and the highest supported version (7.4) was enough. Do you agree?

**Comment by javiereguiluz** [comment]
I need help to debug this issue in Travis. Thanks!

```
$ [[ $TRAVIS_PHP_VERSION != "7.4" ]] || ./vendor/bin/php-cs-fixer fix --diff --dry-run -v
PHP Fatal error:  Uncaught Error: Call to undefined method PhpCsFixer\Config::create() in /home/travis/build/symfony/demo/.php_cs.dist:23
Stack trace:
#0 /home/travis/build/symfony/demo/vendor/friendsofphp/php-cs-fixer/src/Console/ConfigurationResolver.php(931): include()
#1 /home/travis/build/symfony/demo/vendor/friendsofphp/php-cs-fixer/src/Console/ConfigurationResolver.php(227): PhpCsFixer\Console\ConfigurationResolver::separatedContextLessInclude('/home/travis/bu...')
#2 /home/travis/build/symfony/demo/vendor/friendsofphp/php-cs-fixer/src/Console/ConfigurationResolver.php(623): PhpCsFixer\Console\ConfigurationResolver->getConfig()
#3 /home/travis/build/symfony/demo/vendor/friendsofphp/php-cs-fixer/src/Console/ConfigurationResolver.php(456): PhpCsFixer\Console\ConfigurationResolver->getFormat()
#4 /home/travis/build/symfony/demo/vendor/friendsofphp/php-cs-fixer/src/Console/Command/FixCommand.php(154): PhpCsFixer\Console\ConfigurationResolver->getReporter()
#5 /home/travis/build/symfony/demo/vendor/symfony/console/Command/Co in /home/travis/build/symfony/demo/.php_cs.dist on line 23
Fatal error: Uncaught Error: Call to undefined method PhpCsFixer\Config::create() in /home/travis/build/symfony/demo/.php_cs.dist:23
Stack trace:
#0 /home/travis/build/symfony/demo/vendor/friendsofphp/php-cs-fixer/src/Console/ConfigurationResolver.php(931): include()
#1 /home/travis/build/symfony/demo/vendor/friendsofphp/php-cs-fixer/src/Console/ConfigurationResolver.php(227): PhpCsFixer\Console\ConfigurationResolver::separatedContextLessInclude('/home/travis/bu...')
#2 /home/travis/build/symfony/demo/vendor/friendsofphp/php-cs-fixer/src/Console/ConfigurationResolver.php(623): PhpCsFixer\Console\ConfigurationResolver->getConfig()
#3 /home/travis/build/symfony/demo/vendor/friendsofphp/php-cs-fixer/src/Console/ConfigurationResolver.php(456): PhpCsFixer\Console\ConfigurationResolver->getFormat()
#4 /home/travis/build/symfony/demo/vendor/friendsofphp/php-cs-fixer/src/Console/Command/FixCommand.php(154): PhpCsFixer\Console\ConfigurationResolver->getReporter()
```

**Comment by rosier** [comment]
I think changing the line https://github.com/symfony/demo/blob/978b34f221ba6973ca8d97f98ee10fb3093d282e/.php_cs.dist#L23 to `return (new PhpCsFixer\Config())` should fix the problem.

For more info see https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4828

**Comment by javiereguiluz** [comment]
@rosier yes!! Thanks a lot for your help. Tests are green again!

**Pull request merged**